src/app.js

```diff
@@ -22,17 +26,31 @@ export function app(app) {
     })
   }
 
-  emit("ready", render(state, view))
+  render()
+
+  emit("init")
 
   return emit
 
-  function render(state, view) {
-    element = patch(
-      root,
-      element,
-      node,
-      (node = emit("render", view)(state, actions))
-    )
+  function render() {
+    if (!locked) {
+      locked = !locked
+
+      requestAnimationFrame(function() {
+        element = patch(
+          root,
+          element,
+          node,
+          (node = emit("render", view)(state, actions))
+        )
+
+        locked = !locked
```

Comment:
> Isn't it the actual expected behavor?
Maybe I should have stated that `patch` never gets called, so the app state and view are out of sync.